src/titiler/core/titiler/core/factory.py

```diff
@@ -439,13 +444,14 @@ def geojson_statistics(
             """Get Statistics from a geojson feature or featureCollection."""
             fc = geojson
             if isinstance(fc, Feature):
-                fc = FeatureCollection(features=[geojson])
+                fc = FeatureCollection(type="FeatureCollection", features=[geojson])
 
             with rasterio.Env(**env):
                 with self.reader(src_path, **reader_params) as src_dst:
                     for feature in fc:
                         data = src_dst.feature(
                             feature.dict(exclude_none=True),
+                            shape_crs=coord_crs or WGS84_CRS,
```

Comment:
we will always default to `WGS84_CRS` (we can't pass `None` because rio-tiler won't like it)

Reply:
small question possibly because I am naive about fastapi, but would it be possible/better to set the coord_crs default value to WGS84_CRS rather than using ors inside the function call?

Reply:
The issue is that if you set a default in the `Query` parameter it will appear in the OpenAPI documentation and be added in the URLs by default which I find annoying (we can see this with the `return_mask` option). 

Also in the future we will want to only rely on rio-tiler's default (right now it won't like if you pass `None`).

Reply:
okay wasn't aware of that, sounds good to me then!